examples/components/data_table/selectable_usage.html

```diff
@@ -0,0 +1,69 @@
+<md-content class="md-padding" layout="row" layout-wrap layout-align="center start">
+
+  <div flex="50" layout="column" flex-xs="100">
+    <md-data-table [selectable]="true">
+      <thead>
+      <tr>
+        <th class="md-data-table__cell--non-numeric">Material</th>
+        <th>Quantity</th>
+        <th>Unit price</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr *ngFor="#material of materials" [selectableValue]="material.id">
+        <td class="md-data-table__cell--non-numeric">{{ material.name }}</td>
+        <td>{{ material.quantity }}</td>
+        <td>{{ material.price }}</td>
+      </tr>
+      </tbody>
+    </md-data-table>
+  </div>
+
+
+  <div flex="100" layout="column" flex-xs="100" layout-align="left start">
```

Comment:
No problem. I will change it but at the moment, you take the content of readme file and insert it in the example/app and the render is not so pretty :)